packages/workers-shared/package.json

```diff
@@ -61,6 +61,7 @@
 		"@cloudflare/workers-tsconfig": "workspace:*",
 		"@cloudflare/workers-types": "^4.20250204.0",
 		"@sentry/cli": "^2.37.0",
+		"@sentry/types": "^7.51.0",
```

Comment:
No need for these types. They are already inferrable from the RewriteFrames type.